ci/aws-analytical-env/shared/meta.yml

```diff
@@ -214,3 +214,46 @@ meta:
       params:
         TF_CLI_ARGS_plan: -lock-timeout=300s
         TF_INPUT: "false"
+
+    vcs-to-s3:
+      task: vcs-to-s3
```

Comment:
can the naming be made more consistent? Some places uses `vcs` and others use `git`

Reply:
Amended code